pkgs/by-name/mu/muffon/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, fetchurl
+, appimageTools
+, makeWrapper
+}:
+
+let
+  pname = "muffon";
+  version = "2.0.2";
```

muffon recently updated to 2.0.3:
```suggestion
  version = "2.0.3";
```